Makefile

```diff
@@ -1,13 +1,13 @@
-GO_VERSION ?= 1.15
-GO_CI_VERSION = v1.31.0
+GO_VERSION ?= 1.18
+GO_CI_VERSION = v1.46.1
 BINARIES = aws-env
 WD ?= $(shell pwd)
 NAMESPACE=sendgrid
 APPNAME=aws-env
 
 export GIT_COMMIT ?= $(shell git rev-parse --verify HEAD)
 export BUILD_DATE ?= $(shell date -u)
-export VERSION ?= $(or $(shell cat version_wf),0.0.1)
+export VER_VERSION ?= 0.0.1
```

Comment:
Does this warrant at least a patch bump or do we generally not increment the shell script?

Reply:
Got it, thank you, this clarifies the use of VER_VERSION for me.